packages/utils/src/types.ts

```diff
@@ -1013,10 +1017,6 @@ export type UiSchema<
     'ui:globalOptions'?: GlobalUISchemaOptions;
     /** Allows the form to generate a unique prefix for the `Form`'s root prefix  */
     'ui:rootFieldId'?: string;
-    /** Allows RJSF to override the default field implementation by specifying either the name of a field that is used
-     * to look up an implementation from the `fields` list or an actual one-off `Field` component implementation itself
-     */
-    'ui:field'?: Field<T, S, F> | string;
```

Comment:
Got it, `MakeUIType` will handle this with `field` defined in `UIOptionsBaseType`